fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeActivity.java

```diff
@@ -262,6 +279,11 @@ public void setupSocialMediaButton() {
     socialMediaButton.setOnClickListener(v -> mViewModel.openSocialMedia());
   }
 
+  public void setupDigitalCashButton() {
+    Button socialMediaButton = findViewById(R.id.tab_digital_cash);
```

Comment:
```suggestion
    Button button = findViewById(R.id.tab_digital_cash);
```

Reply:
Maybe even change to `digitalCashButton` or `trasactionButton `is even clearer